src/bundle/Resources/views/form_fields.html.twig

```diff
@@ -37,10 +37,10 @@
 {%- endblock -%}
 
 {%- block _search_content_types_widget -%}
-    <div class="ez-content-type-selector ez-content-type-selector--collapsed">
+    <div class="ez-content-type-selector">
     {%- set options = choices -%}
     {%- for group_label, choice in options -%}
-        <div class="ez-content-type-selector__group">
+        <div class="ez-content-type-selector__group {% if not loop.first %}ez-content-type-selector__group--collapsed{% endif %}">
```

Comment:
So we will still have this collapsed content types?

Reply:
Yes, as you can see on screens in PR description and prototypes. :)

Reply:
But we no longer display them in the popup-like `div`.